cds-snc/cppd-medical-report
# Section 2 French Translation

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates section 2 of the medical report.

# Test Instructions

1. Go to the application
1. If your browser is in EN-xx locale, click `Français` at the top
1. Navigate through the intro section and Section 1 to get to Section 2
1. See French translation (original EN and FR forms are linked in the Trello card)
